oneanddone/tasks/models.py

```diff
@@ -517,6 +524,14 @@ def users_who_completed_this_task(self):
                 users.append(user)
         return users
 
+    @property
+    def total_votes(self):
```

Comment:
Sometimes this is called `rating` and sometimes you use the term `votes`, which is confusing. I think `rating` makes more sense, so I suggest you use the term `rating` everywhere and do away with `votes`.